src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -861,7 +862,7 @@ namespace AppInstaller::CLI::Workflow
     void GetManifestWithVersionFromPackage::operator()(Execution::Context& context) const
     {
         PackageVersionKey key("", m_version, m_channel);
-        auto requestedVersion = context.Get<Execution::Data::Package>()->GetAvailableVersion(key);
+        auto requestedVersion = context.Get<Execution::Data::Package>()->GetAvailableVersion(key, PinBehavior::ConsiderPins);

```

Comment:
```suggestion
        auto requestedVersion = context.Get<Execution::Data::Package>()->GetAvailableVersion(key, PinBehavior::IgnorePins);

```

This looks to be only used by show command

Reply:
The upgrade command also uses this. I added an argument to the workflow task to consider or not the pins so that `show` can ignore them.